### PR TITLE
Add resource_path function

### PR DESCRIPTION
**Pull request by matthijs110**
This function does exists. But not in the documentation..

### CODE REVIEW

File: helpers.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Laravel includes a variety of "helper" PHP functions. Many of these functions ar
 [elixir](#method-elixir)
 [public_path](#method-public-path)
 [storage_path](#method-storage-path)
+[resource_path](#method-resource_path)
```
Comment (taylorotwell) [inline helpers.md]: Please put in alphabetical order.

Comment (matthijs110) [inline helpers.md]: Done. :)

File: helpers.md [hunk 2]
```diff
@@ -456,6 +457,17 @@ The `public_path` function returns the fully qualified path to the `public` dire
...
+<a name="method-resource-path"></a>
+#### `resource_path()` {#collection-method}
+
+The `resource_path` function returns the fully qualified path to the `resources` directory:
+
+    $path = resource_path();
+
+You may also use the `resource_path` function to generate a fully qualified path to a given file relative to the storage directory:
+
+    $path = storage_path('assets/sass/app.scss');
```
Comment (arthurkirkosa) [inline helpers.md]: Should this be referencing ```resource_path``` instead of ```storage_path```?

Comment (matthijs110) [inline helpers.md]: Not instead of. It just references the resource folder.

**Pull request merged**